### ISSUE TITLE
Sapper swallows Rollup error locations

### ISSUE THREAD

**Issue by benmccann** [sveltejs/sapper#1234]
**Describe the bug**
Sapper discards the file name, line number, etc. when printing error messages

**Logs**
Please include browser console and server logs around the time this bug occurred.

**To Reproduce**
```
git clone git@github.com:babichjacob/sapper-typescript-graphql-template.git
mv src/service-worker.js src/service-worker.ts
npm install
npm run build
```

The output does not include a filename or line number:
```
> @rollup/plugin-typescript TS6133: 'routes' is declared but its value is never read.
```

**Expected behavior**

Include the details that are present in the error object:
```
{"pluginCode":"TS6133","loc":{"column":35,"line":1,"file":"src/service-worker.ts"},"frame":"\n\u001b[7m1\u001b[0m import { timestamp, files, shell, routes } from '@sapper/service-worker';\n\u001b[7m \u001b[0m \u001b[91m                                  ~~~~~~\u001b[0m\n","code":"PLUGIN_ERROR","plugin":"typescript","hook":"buildStart"}
```

**Stacktraces**
Sapper does not provide a stack trace. However, on the line below the `err` object does contain additional details that get discarded at some point.

https://github.com/sveltejs/sapper/blob/b80346e2a6b6a4eded87b5c4aabafd0c7d941f44/src/core/create_compilers/RollupCompiler.ts#L73

**Information about your Sapper Installation:**
- Sapper version: `master`

**Severity**
Critical. Developing an application in Sapper is nearly impossible

**Additional context**
Please note that this bug has a very different root cause than https://github.com/sveltejs/sapper/issues/1221

I got this when trying to develop a Sapper application based off @babichjacob's TypeScript template

**Comment by benmccann** [comment]
I think this is probably the relevant section of code from Rollup:
https://github.com/rollup/rollup/blob/0c4a5db25ce3c0b7f55390b174e0a1c9901f8d24/cli/logging.ts#L14

However, I question the strategy of rebuilding Rollup inside Sapper. It'd probably be nicer if we just executed Rollup and avoided the potential for these types of issues

**Comment by Conduitry** [comment]
Fixed in 0.27.14.
